config/prod.exs

```diff
@@ -23,4 +23,5 @@ config :logger, level: :info
 # Newrelic agent
 config :new_relic_agent,
   app_name: System.get_env("NEWRELIC_APP_NAME"),
-  license_key: System.get_env("NEWRELIC_KEY")
+  license_key: System.get_env("NEWRELIC_KEY"),
+  disable_erlang_trace: if(System.get_env("FLAMEGRAPH"), do: true, else: false)
```

Comment:
What does this do?

Reply:
flame_on uses tracing to generate its output, new relic also uses tracing for some of its metrics, but there can only be one running tracing process, see this [issue](https://github.com/newrelic/elixir_agent/issues/137).